common/source/docs/common-blheli32-passthru.rst

```diff
@@ -28,4 +30,10 @@ To enable BLHeli pass-through you need to set the following parameters and reboo
 Now connect a USB cable to your autopilot and use BLHeliSuite32 on Windows to connect. Select "BLHeli32 Bootloader (Betaflight/Cleanflight)" from the interfaces menu.
 
 .. image:: ../../../images/blhelisuite32.jpg
-    :target: ../_images/blhelisuite32.jpg
\ No newline at end of file
+    :target: ../_images/blhelisuite32.jpg
+
+BLHeli32 ESC Telemetry
+----------------------
+
+While many ESCs offer ESC telemetry (RPM,Current,etc.) reporting via a separate serial signal lead (see: :ref:`ESC Telemetry <esc-telemetry>`, some BLHeli32 ESCs offer telemetry reporting over the control link. See :ref:`common-dshot-blheli32-telemetry`.
```

Comment:
Only RPM is available, this type of telemetry is specifically for rpm filtering

Reply:
Which? over single wire  is limited or BLHeli3 or both..thought you could get temp, voltage, current and rpm from both....original docs show a single wire telemetry connectioin in place of a battery monitor for voltage and current

Reply:
Telemetry reporting over the control link on BLHeli32 only supports RPM